src/Sulu/Component/Content/Types/ResourceLocator/Strategy/TreeFullEditStrategy.php

```diff
@@ -303,4 +243,26 @@ public function deleteByPath($path, $webspaceKey, $languageCode, $segmentKey = n
     {
         $this->mapper->deleteByPath($path, $webspaceKey, $languageCode, $segmentKey);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getChildPart($resourceSegment)
+    {
+        $divider = strrpos($resourceSegment, '/');
+
+        if ($divider === false) {
+            return $resourceSegment;
+        }
+
+        return substr($resourceSegment, $divider + 1);
```

Comment:
Couldn't that be implemented more easily? It should just remove the starting `/`, shouldn't it?